bundles/org.eclipse.equinox.ds.tests/.gitignore

```diff
@@ -1 +1,2 @@
 /scr_test
+OSGI-INF/org.eclipse.equinox.*.xml
```

Comment:
```suggestion
OSGI-INF/
```

Reply:
I reduced it now to `OSGI-INF/org.eclipse.*.xml`, ignoring all XML could be too restrictive. And since in the fast majority of cases the classes in eclipse projects are in a `org.eclipse...` package this should be fine. And if not, one will notice it quickly and can adjust the pattern in that specific case.